homework/zadanie4_five_philosophers.cpp

```diff
@@ -55,13 +35,17 @@ class CPhilosoph{
         std::scoped_lock sl(left_fork_->mtx_fork_, right_fork_->mtx_fork_);
         output_ = "Philosoph: " + name_ + " is eating...\n";
         std::cout<<output_;
-        std::this_thread::sleep_for(2s); 
+        std::mt19937 gen(rd());
+        std::uniform_int_distribution<> dis(1, 3);
```

Comment:
Ponieważ tego generatora używasz w kilu miejscach to przenieś to do konstruktora.

Reply:
przeniesiono do konstruktora,